src/hosted-form/iframe-content/card-number-formatter.ts

```diff
@@ -0,0 +1,39 @@
+import { number } from 'card-validator';
+import creditCardType from 'credit-card-type';
+import { max } from 'lodash';
+
+export default class CardNumberFormatter {
+    constructor(
+        private _separator: string = ' '
```

Comment:
Just wondering why we need this in the constructor?

Reply:
I guess I don't really need it. The separator could just be a hard coded constant.

Reply:
I've updated the code and removed the parameter from the constructor.